pkg/rotator/rotator.go

```diff
@@ -45,14 +45,40 @@ const (
 
 var crLog = logf.Log.WithName("cert-rotation")
 
-var vwhGVK = schema.GroupVersionKind{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: "ValidatingWebhookConfiguration"}
+var (
+	vwhGVK = schema.GroupVersionKind{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: "ValidatingWebhookConfiguration"}
+	mwhGVK = schema.GroupVersionKind{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: "MutatingWebhookConfiguration"}
+	crdGVK = schema.GroupVersionKind{Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: "CustomResourceDefinition"}
+)
+
+//WebhookType it the type of webhook, either validating/mutating webhook or a CRD conversion webhook
+type WebhookType int
 
-var crdGVK = schema.GroupVersionKind{Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: "CustomResourceDefinition"}
+var (
+	//ValidatingWebhookType indicates the webhook is a ValidatingWebhook
+	ValidatingWebhookType WebhookType
+	//MutingWebhookType indicates the webhook is a MutatingWebhook
+	MutingWebhookType WebhookType = 1
+	//CRDWebhookType indicates the webhook is a conversion webhook
+	CRDConversionWebhookType WebhookType = 2
+)
```

Comment:
Go standard is use use consts for these:

```
const (
  ValidatingWebhook WebhookType = iota
  MutatingWebhook
  CRDConversionWebhook
)
```

If you're not sure about what `iota` is, [see here](https://golang.org/ref/spec#Iota).

I'd also drop the `Type` suffix. Outside this package, I think `rotator.ValidatingWebhook` is pretty clear and I don't think appending `Type` to it adds much. Inside the package, there shouldn't be any conflicts with private names. But I don't feel too strongly about it, just a suggestion.

Reply:
Done